src/site/cookies/index.njk

```diff
@@ -0,0 +1,101 @@
+---
```

Comment:
I was hoping to reuse the `markdown-page.njk` template here but because of the notification banner I couldn't.

Reply:
Hmm is it something that could be accomplished with a block?